draft-ietf-quic-recovery.md

```diff
@@ -348,14 +348,22 @@ latency at the peer or loss of previous ACK frames.  Any delays beyond the
 peer's max_ack_delay are therefore considered effectively part of path delay and
 incorporated into the smoothed_rtt estimate.
 
+The first ACK frame in the Handshake and ApplicationData packet number spaces
+can be greatly delayed due to a lack of decryption keys at the time they're
+received. For these two ACK frames, the max_ack_delay MAY be ignored if the
+sender chooses. If the delayed ACK frame is the first RTT sample, ignoring
+max_ack_delay has no impact.
+
 When adjusting an RTT sample using peer-reported acknowledgement delays, an
 endpoint:
 
 - MUST ignore the ACK Delay field of the ACK frame for packets sent in the
-  Initial and Handshake packet number space.
+  Initial and Handshake packet number space, except for the first ACK frame
+  received in Handshake.
```

Comment:
I think that we are again talking about packets that might have been buffered, not just the first.